drawBot/drawBotDrawingTools.py

```diff
@@ -338,7 +340,8 @@ def newPage(self, width: str | float | None = None, height: float | None = None)
         self._dummyContext = DummyContext()
         self._addInstruction("newPage", width, height)
 
-    def pages(self) -> tuple[DrawBotPage, ...]:
+
+    def pages(self) -> tuple["DrawBotPage", ...]:
```

Comment:
Perhaps you can use `from __future__ import annotations` instead? That defers evaluation of the annotations, so you don't have to quote the type.

Reply:
good idea!